### DISCUSSION TITLE
Why I cannot add feature to LTS version?

### DISCUSSION THREAD

**Discussion by melihovv** [symfony/symfony#46920] [Q&A] [answered]
I use symfony 5.4 and want to propose a new feature. According to https://symfony.com/doc/current/contributing/code/pull_requests.html I must add new feature to 6.1 branch. Why I cannot add it to 5.4? I'm not going to upgrade from 5.4 in near future, but I want to use my new feature.

**Comment by GromNaN** [comment, marked as answer]
This is for stability. Any new feature is potentially a breaking change for someone: adding a method to a class, a new option, a log message ...

- Fabien wrote [Stability over Feature](http://fabien.potencier.org/about-symfony-stability-over-features.html) in 2013.
- The [maintenance page](https://symfony.com/doc/current/contributing/code/maintenance.html) is a good documentation on this topic.
